src/site/antora/modules/ROOT/pages/faq.adoc

```diff
@@ -122,15 +122,15 @@ You can set the logging level on the `AppenderRef` element.
 . Make sure you have xref:manual/installation.adoc[the right JAR files] on your classpath.
 
 . Check the name of your configuration file.
-By default, Log4j looks for a configuration file named `log4j2.xml` on the classpath.
+Log4j looks for a configuration file named `log4j2.xml` on the classpath by default.
 Note the `2` in the file name!
 (See xref:manual/configuration.adoc[the Configuration page] for more details.)
 
 . Increase the logging verbosity of the internal xref:manual/status-logger.adoc[]: +
 `-Dlog4j2.statusLoggerLevel=TRACE`
 
 . Enable all internal debug logging: `-Dlog4j2.debug`.
-This disables level-based xref:manual/status-logger.adoc[] filtering and effectively allows all status logs.
+This turns-off level-based xref:manual/status-logger.adoc[] filtering and effectively allows all status logs.
```

Comment:
We technically always use `enable` and `disable` to denote activation – I think we should stick to that.

Reply:
Disabled people will thank us for not using this terminology. We can keep it, but I think it is a negative wording. Please confirm you want to do that.

Reply:
> Disabled people will thank us for not using this terminology.

You mean "people with disabilities" :wink:? I don't think anybody thinks about physical disabilities, when reading a technical documentation, but if we are going to ban the `disable` verb, we should do it in the entire documentation.

Reply:
> Please confirm you want to do that.

@grobmeier, I confirm that *I prefer* replacing `turns-off` with `disables`. Though I leave it to your discretion.

Reply:
How about "This allows for all kinds of status log messages to be written" or something like that.